src/app/work-item/work-item-board/work-item-board.component.scss

```diff
@@ -86,8 +84,12 @@
       height: 100%;
     }
   }
-.side-panel-container {
-  background-color: $color-pf-black-900;
+.side-panel-wrap {
+  min-width: 320px;
+  max-width: 320px;
```

Comment:
@mindreeper2420 only width is not working because of flex. I just did quick fix for this rather than changing whole layout